core/trino-main/src/main/java/io/trino/operator/Driver.java

```diff
@@ -512,10 +512,10 @@ private void destroyIfNecessary()
         try {
             inFlightException = closeAndDestroyOperators(activeOperators);
             if (driverContext.getMemoryUsage() > 0) {
-                log.error("Driver still has memory reserved after freeing all operator memory.");
+                log.debug("Driver still has memory reserved after freeing all operator memory.");
```

Comment:
isn't it an erroneous situation that should be fixed?
if it's not reported, how do we know about this

(same in couple different places)

Reply:
See above: https://github.com/trinodb/trino/pull/19554#issuecomment-1790828066